pkg/service/sync.go

```diff
@@ -25,17 +25,24 @@ type SyncService struct {
 func (s *SyncService) SyncSessionSymbols(ctx context.Context, exchange types.Exchange, startTime time.Time, symbols ...string) error {
 	for _, symbol := range symbols {
 		log.Infof("syncing %s %s trades...", exchange.Name(), symbol)
-		if err := s.TradeService.Sync(ctx, exchange, symbol, startTime); err != nil {
+		// TODO: bbgo import cycle error
+		//markets, err := bbgo.LoadExchangeMarketsWithCache(ctx, exchange)
```

Comment:
you can pull out this `LoadExchangeMarketsWithCache` to pkg/cache

Reply:
pkg/cache or pkg/service/cache ?

Reply:
refactored move cache.go & home.go to pkg/cache